### DISCUSSION TITLE
r/city-china-zhejiang-hangzhou-qingshancun

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#13] [Announcements]
# r/city-china-zhejiang-hangzhou-qingshancun

游记 | 一个充满活力的乌托邦

https://blog.ops-coffee.cn/r/city-china-zhejiang-hangzhou-qingshancun

**Comment by esuo** [comment]
刚刚参加工作 希望几年之后也能有这般对生活的热爱

**Reply by ops-coffee** [reply]
无论何时，都不要放弃对生活的热爱

**Comment by cordream** [comment]
很喜欢你分享的打卡游记 ，感受到浓浓的生活热爱。。

**Reply by ops-coffee** [reply]
很高兴你能喜欢
